ArkhamOverlay/Utils/IHasImageButton.cs

```diff
@@ -27,19 +27,26 @@ public static class HasImageButtonExtensions {
             }
         }
 
-        private static void DoLoadImage(IHasImageButton hasImageButton, string url) {
+        private static void DoLoadImage(IHasImageButton hasImageButton, string uri) {
```

Comment:
I'd add a try-catch somewhere around this method since the constructors for BitmapImage and Uri can both throw.

Reply:
We'll move this into a service so we have access to a logger

Reply:
Added a todo